lib/net.c

```diff
@@ -906,6 +912,9 @@ init_iface(struct gatekeeper_if *iface)
 	}
 
 	/* Initialize all ports on this interface. */
+
+	gatekeeper_port_conf.rxmode.max_rx_pkt_len = iface->mtu;
```

Comment:
Commit a60204254c1450d8f17453e0077075661e659410 already modifies `gatekeeper_port_conf.rx_adv_conf.rss_conf.rss_hf` in `init_port()`, and now this patch modifies `gatekeeper_port_conf.rxmode.max_rx_pkt_len`. `gatekeeper_port_conf` should (1) become a constant, and (2) be renamed to `gatekeeper_template_port_conf`. Whenever it's used, a local variable is initialized with the template. It won't impact performance because the copy only happens during initialization. However, this change will make sure we don't silently carry a change in the template from a place to another; this would be a hard bug to catch.